x/gov/client/cli/tx.go

```diff
@@ -3,20 +3,19 @@ package cli
 import (
 	"fmt"
 	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
 
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/client/utils"
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/version"
 	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
-	"github.com/cosmos/cosmos-sdk/x/gov"
-
-	"strings"
-
-	"github.com/spf13/cobra"
-
-	govClientUtils "github.com/cosmos/cosmos-sdk/x/gov/client/utils"
+	govutils "github.com/cosmos/cosmos-sdk/x/gov/client/utils"
```

Comment:
I like the idea of `gcutils` appears in another place in this PR.
Maybe we that could be the rule to use for long aliases?

Reply:
although I like that it's one character shorter, I find that acronym move confusing that `govutil`, so I'm leaving it as is

Reply:
yes, my comment was not about length but about using consistent rules in all the codebase.

Reply:
@jleni I'm not sure an _explicit_ consistent ruleset is possible, maybe a loose one... If you feel driven to, why not make a PR to a new section in `contributing.md` - we could define a high level stylistic loose suggested ruleset there! lmn - resolving for now